signhandler/src/main/java/se/swedenconnect/signservice/signature/tbsdata/impl/AbstractTBSDataProcessor.java

```diff
@@ -60,6 +63,17 @@ public abstract class AbstractTBSDataProcessor implements TBSDataProcessor {
   /** Supported processing rules */
   private final List<String> supportedProcessingRules;
 
+  /** The clock skew that we accept during checks of time stamps. */
+  @Setter
+  protected Duration allowedClockSkew = ValidationConfiguration.DEFAULT_ALLOWED_CLOCK_SKEW;
```

Comment:
Could be private.

Reply:
I know, but I'm not sure if this isn't valid data for any future implementations of this abstract class. It seemed reasonable to me to keep it as protected.